### ISSUE TITLE
1.2 Release

### ISSUE THREAD

**Issue by Thom1729** [SublimeText/sublime_lib#62]
The following new features have been merged since 1.1:

- `show_selection_panel`.
- `new_window` and `close_window`.
- Improved `ViewStream.seek`.
- `ViewStream.show_cursor`, etc.
- `ResourcePath` (#52)

~~`ResourcePath` seems to be nearly there.~~
Other minor features/improvements we might make:

- [x] Adding a convenience function to create `IntFlag`s from lists of strings. (#63)
- [x] Adding a `ResourcePath.from_file_path` method. (#66)
- [x] Cleaning up `__init__.py`. (#64)
- [x] Factor out non-exported utility functions into a `_util` directory. (#68)
- [x] Improve formatting of comments. (#77)
- [x] Improve test coverage. (#65)

Any/all of these could probably be done in a week with very little revision.

**Comment by FichteFoll** [comment]
Is that supposed to be a milestone/release tracking issue? Otherwise, I created #56 to notify people about changes.

Although GitHub did very recently add the feature to only follow a repo's releases (and not all issues), so we'd only need to use #56 for breaking changes, i.e. major versions.

---

I agree with this list. I'll add a short checklist to #52 so we can get that merged fast and then proceed with this list. I also turned your lists into checklists.
